install/action.yaml

```diff
@@ -50,35 +57,35 @@ runs:
       with:
         fetch-depth: "100"
     - run: git fetch --prune --tags --unshallow
-      shell: bash -l {0}
+      shell: bash -el {0}
     - uses: actions/setup-python@v4
       with:
         python-version: ${{ inputs.python-version }}
     - uses: conda-incubator/setup-miniconda@v2
       with:
-        miniconda-version: "latest"
-    - if: inputs.conda-update == 'true'
-      run: |
-        conda update -c defaults -n base conda
-      shell: bash -l {0}
+        miniforge-variant: Mambaforge
+        channels: conda-forge
```

Comment:
Just noting that effectively means that we're no longer using *defaults* at all.

Reply:
After thinking a bit about this, we could make this depend on the `conda-mamba` input. I think this will answer your three comments.

Reply:
I went back to using miniconda and then installed mamba in a separate step.